.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: 'GitHub CI'
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
we currently support node 8. I'm down with just supporting the current LTSes, tho 👍

Reply:
Ah. I didn't realize that. I like the idea of supporting only current LTS, but what does Jest itself officially support? Or is that what you meant by "we"?